README.md

```diff
@@ -12,23 +12,14 @@ Esto posibilita tener un backup de los gastos en la nube, a la vez que poder rea
 ## Planificación
 Hemos planificado [4 milestones (estructura y API)](https://github.com/marcosrmartin/PerroAndaluz/milestones) los cuales describen unos productos mínimamente viables, y [2 historias de usuarios](https://github.com/marcosrmartin/PerroAndaluz/issues) los cuales expresan funcionalidades que al cliente le gustaría obtener.
 
-### Milestones
-#### [M-0] Estructura de datos para gestionar las cuentas de cada usuario
-Definición de un módulo que permita guardar los datos relacionados con las listas y que el mismo compile sin problemas.
+## Elección del gestor de dependencias
```

Comment:
No pongáis esto en el README, no forma parte de lo que debe leer quien llegue al proyecto.